persistence/scheduled-tasks/schedule-task-via-schtasks.yml

```diff
@@ -1,6 +1,6 @@
 rule:
   meta:
-    name: schedule task via command line
+    name: schedule task via schtasks
```

Comment:
should we leave this as `command line` as we also include `Register-ScheduledTask`?

Reply:
im not sure, since then "at.exe" would fall back into this rule. i think this is probably fine as is, unless we run into problems or confusion?

Reply:
yeah, so we're good to merge?